tests/format.rs

```diff
@@ -0,0 +1,46 @@
+#[cfg(test)]
+mod tests {
+    use yaiwr::{err::InterpError, scope::Scope, Calc};
+
+    pub fn eval_prog(
+        calc: &mut Calc,
+        input: &str,
+        scope: &mut Scope,
+    ) -> Result<Option<u64>, InterpError> {
+        let ast = calc.from_str(input).unwrap();
+        let bytecode = Calc::ast_to_bytecode(ast);
+        return calc.eval(&bytecode, scope);
+    }
+
+    #[test]
+    fn multiline_function_calls() {
+        let calc = &mut Calc::new();
+        let scope = &mut Scope::new();
+        eval_prog(
+            calc,
+            "  
+            fun f3(){ 
+                println(2);
+            }  
+            fun f2(_a, _b){ 
+                f3();
+                return _a + _b;
+            }
+            fun f1(_a, _b){ 
+                return f2(_a, 10); + _b;
```

Comment:
This syntax is odd. Is this because a function call requires a terminating `;`?

Reply:
Yes, I agree. I intentionally didn't fix it in this PR. But I added it as a TODO task for future

Reply:
ok